WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewModel.swift

```diff
@@ -89,11 +136,12 @@ extension ReaderTabViewModel {
             guard let topics = try ContextManager.sharedInstance().mainContext.fetch(topicsFetchRequest) as? [ReaderAbstractTopic] else {
                 return
             }
-
-            completion(ReaderHelpers.rearrange(items: topics.map { ReaderTabItem(topic: $0) }))
+            let items = ReaderHelpers.rearrange(items: topics.map { ReaderTabItem(topic: $0) })
```

Comment:
Do we need the extra `items` variable here? Could we just use `tabItems` directly?

Reply:
I'm gonna change this a bit in the next PR, so please bear with me :)